recipes/pywin32/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "pywin32" %}
+{% set version = "220" %}
+{% set hash_type = "sha1" %}
+{% set hash = "88ba6a31c077a9e1b49b59dc215ddadc6aa5e3d6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://downloads.sourceforge.net/project/{{ name }}/{{ name }}/Build {{ version }}/{{ name }}-{{ version }}.zip
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup{{ '3' if py3k else '' }}.py install --record=record.txt
```

Comment:
As another option, which may be a bit cleaner, one can have a multiline script with selectors. Actually now that I think of it `pyinstaller` has a good example of this. 😆